python_modules/airline-demo/environments/local_full_download.yml

```diff
@@ -0,0 +1,50 @@
+solids:
+  download_archives:
```

Comment:
We can do this in a separate diff, but should we model these an inputs and make skip_if_present a solid-wide config?

Reply:
Yeah, probably -- I'd like to document the current state of the repo and then make some improvements to the codebase.